lib/Runtime/Library/JavascriptNumber.cpp

```diff
@@ -293,6 +294,11 @@ namespace Js
             // pow([+/-] 1, Infinity) = NaN according to javascript, but not for CRT pow.
             return JavascriptNumber::NaN;
         }
+        else if (TryGetInt32Value(y, &intY))
```

Comment:
Thanks. Just run though Octane, Kraken and SunSpider again last build from master. No regression detected.

---
In reply to: [73776762](https://github.com/Microsoft/ChakraCore/pull/1396#discussion_r73776762) [](ancestors = 73776762)